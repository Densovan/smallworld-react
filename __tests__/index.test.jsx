import React from "react"
import { shallow } from "enzyme"
import Index from "../src/components/index"

describe("Index component", () => {
    it("Index renders without crashing", () => {
        const wrapper = shallow(<Index/>)
        expect(wrapper).toMatchSnapshot()
    })
})