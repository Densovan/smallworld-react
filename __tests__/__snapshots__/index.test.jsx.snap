// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index component Index renders without crashing 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      SmallWorld Venture | Homegrown startup community based in Phnom Penh
    </title>
    <meta
      content="Homegrown Startup Community. We began in 2011 by providing a collaborative workspace environment for entrepreneurs, and then quickly moved forward raising investment capital to fund new startup projects."
      name="description"
    />
  </HelmetWrapper>
  <div>
    <div
      className="backBanner"
      style={
        Object {
          "backgroundImage": "url(\\"/images/banner.png\\")",
          "height": "54vw",
        }
      }
    >
      <div>
        <div
          className=""
        >
          <div
            className="ui left demo vertical inverted labeled icon sidebar menu overlay visible mobile only slideOut"
            style={Object {}}
          >
            <Link
              className="item"
              to="/about-us"
            >
              About
            </Link>
            <Link
              className="item"
              to="/what-we-do"
            >
              Works
            </Link>
            <Link
              className="item"
              to="/news-and-events"
            >
              News
            </Link>
            <Link
              className="item"
              to="/contact-us"
            >
              Contact
            </Link>
          </div>
        </div>
        <div
          className="hd"
        >
          <div
            className="ui secondary container menu mobile only"
          >
            <div
              className="menu left"
            >
              <Link
                to="/"
              >
                <img
                  alt="SmallWorld Venture"
                  className="logonav"
                  src="/images/logo/sw-green.png"
                />
              </Link>
            </div>
            <div
              className="menu right asize"
            >
              <img
                alt=""
                className="menu-icons"
                height="30px"
                onClick={[Function]}
                src="/images/menu.svg"
              />
            </div>
          </div>
          <div
            className="ui secondary container menu mobile hidden"
          >
            <div
              className="menu left"
            >
              <Link
                to="/"
              >
                <img
                  alt="SmallWorld Venture"
                  className="logonav"
                  src="/images/logo/sw-green.png"
                />
              </Link>
            </div>
            <div
              className="menu right asize"
            >
              <Link
                className="item"
                to="/about-us"
              >
                About
              </Link>
              <Link
                className="item"
                to="/what-we-do"
              >
                Works
              </Link>
              <Link
                className="item"
                to="/news-and-events"
              >
                News
              </Link>
              <Link
                className="item"
                to="/contact-us"
              >
                Contact
              </Link>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ui container"
      >
        <div
          className="bannerContent"
        >
          <h3>
            <span>
              &lt;
            </span>
             
            Homegrown Startup Community
             
            <span>
              /&gt;
            </span>
          </h3>
          <p
            className="paragraph"
          >
            We began in 2011 by providing a collaborative workspace environment for entrepreneurs, and then quickly moved forward raising investment capital to fund new startup projects.
          </p>
          <p
            className="paragraph"
          >
            With a variety of research and development projects in motion today, we're involved in startup venture building through community supported seed equity investments, together with rural ecovillage development and long-term land management.
          </p>
          <Link
            to="/about-us"
          >
            <button
              className="ui basic button btnLearnMore"
            >
              LEARN MORE
            </button>
          </Link>
        </div>
      </div>
    </div>
    <div
      className="cvcolor"
    >
      <div
        className="ui container margin-top"
      >
        <div
          className="ui four column doubling stackable grid"
        >
          <div
            className="column "
          >
            <center>
              <div
                className="newsDetailo"
              >
                <img
                  alt="SmallWorld Venture"
                  className="imageIndex"
                  src="/images/blurb-1.png"
                />
                <div
                  className="shadowIndex"
                >
                  <h4>
                    Startup Community
                  </h4>
                  <p>
                    SmallWorld Ventures is committed to becoming the number one catalyst for funding and assisting startups in Cambodia.
                  </p>
                  <br />
                  <br />
                  <br />
                  <br />
                </div>
              </div>
            </center>
          </div>
          <div
            className="column"
          >
            <center>
              <div
                className="newsDetailo"
              >
                <img
                  alt="SmallWorld Venture"
                  className="imageIndex"
                  src="/images/blurb-2.png"
                />
                <div
                  className="shadowIndex"
                >
                  <h4>
                    Seed Equity Investments
                  </h4>
                  <p>
                    Seed Equity Investments are provided to promising startup teams with projects ranging between 5,000 to 25,000 USD.
                  </p>
                  <br />
                  <br />
                  <br />
                  <br />
                </div>
              </div>
            </center>
          </div>
          <div
            className="column"
          >
            <center>
              <div
                className="newsDetailo"
              >
                <img
                  alt="SmallWorld Venture"
                  className="imageIndex"
                  src="/images/blurb-3.png"
                />
                <div
                  className="shadowIndex"
                >
                  <h4>
                    Venture Building
                  </h4>
                  <p>
                    Smallworld Ventures has built an internal team of technicians engaged in research and development projects with an aim to spin off new ventures.
                  </p>
                  <br />
                  <br />
                  <br />
                </div>
              </div>
            </center>
          </div>
          <div
            className="column"
          >
            <center>
              <div
                className="newsDetailo"
              >
                <img
                  alt="SmallWorld Venture"
                  className="imageIndex"
                  src="/images/blurb-4.png"
                />
                <div
                  className="shadowIndex"
                >
                  <h4>
                    Ecovillage Development 
                  </h4>
                  <p>
                    At our rural ecovillage project, we're building a hands-on learning, working, and living environment with a balance among the natural world, education, economics, and sustainable living practices.
                  </p>
                  <br />
                  <br />
                </div>
              </div>
            </center>
          </div>
        </div>
      </div>
    </div>
    <div
      className="backInformation"
    >
      <div
        className="ui container"
      >
        <div
          className="ui stackable two column grid"
        >
          <div
            className="column"
          >
            <h1>
              Have some ideas for new venture?
            </h1>
            <p>
              We're looking for new approaches to problem solving and creating business.Do you have an innovative idea for a startup venture?
            </p>
            <p>
              Or maybe you have a skillset in mind that you want to develop as you work within our existing SmallWorld venture.
            </p>
            <p>
              We're open to discussing your own startup ideas, however unconventional, regardless of age, gender, or nationality.
            </p>
            <p>
              Send us a message!
            </p>
            <Link
              to="/contact-us"
            >
              <button
                className="ui basic button btnInformation"
              >
                CONTACT US
              </button>
            </Link>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ui container margin-buttons"
    >
      <h2
        className="newsAndEvent"
        style={
          Object {
            "marginTop": "60px",
          }
        }
      >
        Community Update
      </h2>
      <div
        className="ui stackable three column equal height stretched grid"
      />
    </div>
    <div
      className="ui container margin-buttons"
    >
      <h2
        className="newsAndEvent"
      >
        KOOMPI News
      </h2>
      <div
        className="ui stackable three column equal height stretched grid"
      />
    </div>
  </div>
  <Footer />
</div>
`;
